src/LimitInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+interface LimitInterface
```

Comment:
Need phpdoc.

Reply:
Yes, that's for the trait check.

Reply:
After discussions in PM it was decided that:

- PHPDoc will be handled as a part of issue #364.
- Postpone interface / trait necessity discussion and possible deletion to a separate issue / PR.